index.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  build: require('./lib/cfn'),
```

Comment:
Should we export `lib.cli.js` here too in case people want to use it in their scripts?